### DISCUSSION TITLE
Can you cut a new release for the 2.5.0 changes so we can pull from pypi?

### DISCUSSION THREAD

**Discussion by j8169** [benbenbang/types-confluent-kafka#224] [Q&A] [answered]
Even if it's just a point release with the old scheme. Thanks

**Comment by benbenbang** [comment]
> Even if it's just a point release with the old scheme. Thanks

Hey, sorry I totally forgot. Let me do a release tmr morning (CET)

**Comment by benbenbang** [comment]
hello @j8169 you can use 1.3.0 for now

**Comment by benbenbang** [comment, marked as answer]
1.3.6 has all you need 🙂
